Gemfile.jruby-1.9.lock

```diff
@@ -132,8 +132,8 @@ GEM
     logstash-codec-graphite (2.0.2)
       logstash-codec-line
       logstash-core (>= 2.0.0.beta2, < 3.0.0)
-    logstash-codec-json (2.1.0)
-      logstash-core (>= 2.0.0.beta2, < 3.0.0)
+    logstash-codec-json (2.1.1)
+      logstash-core (>= 2.0.0, < 3.0.0)
```

Comment:
Yes, I think this is the case.  Not sure why the dependency changed.
Also this should be `logstash-codec-json 2.1.1` and no 2.1.0 ?

https://rubygems.org/gems/logstash-codec-json/versions/2.1.1

2.1.1 was release before my mass update, any reason why we arent picking up the latest version?